modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -45,6 +45,29 @@ type FileServer struct {
 	// or current working directory otherwise.
 	Root string `json:"root,omitempty"`
 
+	// A list of files or folders to show; the file server will pretend as if
+	// only these exist, hiding everything else. If not set, everything is shown
+	// by default. Accepts globular patterns like `*.ext` or `/foo/*/bar` as
+	// well as placeholders. Because site roots can be dynamic, this list uses
+	// file system paths, not request paths. To clarify, the base of relative
+	// paths is the current working directory, NOT the site root.
+	//
+	// Entries without a path separator (`/` or `\` depending on OS) will match
+	// any file or directory of that name regardless of its path. To show only a
+	// specific file with a name that may not be unique, always use a path
+	// separator. For example, to show all files or folder trees named "shown",
+	// put "shown" in the list. To hide only ./shown, put "./shown" in the list.
+	//
+	// When possible, all paths are resolved to their absolute form before
+	// comparisons are made. For maximum clarity and explictness, use complete,
+	// absolute paths; or, for greater portability, use relative paths instead.
+	//
+	// When both `show` and `hide` are used together, shown files or folders are
+	// checked first (hiding anything not matching), and `hide` will be checked
+	// afterwards, further restrict the results. So, a shown directory can have
+	// files within it hidden by `hide`.
+	Show []string `json:"show,omitempty"`
```

Comment:
Instead of copying so much documentation, can we just explain that this works the same as Hide, and to refer to Hide's docs, except that this lists files to show.

(Or put the docs on Show and have Hide reference Show. Either way.)